pootle/apps/pootle_store/receivers.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+
+from pootle_store.models import Store
+
+
+@receiver(pre_save, sender=Store)
+def store_unit_pre_save_handler(sender, instance, **kwargs):
```

Comment:
@phlax, I'm not sure how `store.pootle_path` can be changed. I think it can't be atm.